modelservice/proxy.py

```diff
@@ -34,14 +35,36 @@ def run(master_hostname="localhost", master_port=50051, flask_port=5000):
     app.run(host="0.0.0.0", port=flask_port)  # Entrypoint
 
 
-@app.route("/model", methods=["GET"])
-def get_model():
+@app.route("/model/<model_id>/gis-join/<gis_join>", methods=["GET"])
+def get_model(model_id, gis_join):
 
-    # TODO Parse query from HTTP JSON -> build gRPC request to Master
+    # Try to cast request data to proper types and return error if there are any issues
+    try:
+        get_model_grpc_request: GetModelRequest = GetModelRequest(
+            model_id=model_id,
+            gis_joins=gis_join
```

Comment:
Lol same thing here. This will have to be updated if you make gis_joins singular in the gRPC message

Reply:
Updated